webapp/src/app.tsx

```diff
@@ -1631,8 +1635,7 @@ ${compileService && compileService.githubCorePackage && compileService.gittag ?
         const tutorialOptions = this.state.tutorialOptions;
         const inTutorial = !!tutorialOptions && !!tutorialOptions.tutorial;
         const docMenu = targetTheme.docMenu && targetTheme.docMenu.length && !sandbox && !inTutorial;
-        const gettingStarted = !sandbox && !inTutorial && !this.state.sideDocsLoadUrl && targetTheme && targetTheme.sideDoc && isBlocks && !targetTheme.useStartPage;
-        const gettingStartedTooltip = lf("Open beginner tutorial");
+        const gettingStarted = !sandbox && !inTutorial && !this.state.sideDocsLoadUrl && targetTheme && targetTheme.sideDoc && isBlocks;
```

Comment:
The gettingStarted button goes away right?

Reply:
Yes no getting started button anymore. However, there is a getting started button on the home screen, and so I'm re-using the logic to determine when that shows.

Reply:
On the home screen, getting started button should always be visible.

Reply:
Okay.